frontend/fingertips-frontend/components/pages/search/results/Results.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+import { expect } from '@jest/globals';
+import { MOCK_DATA } from '@/app/search/results/search-result-data';
+import SearchResults from '.';
+
+describe('Search Results Suite', () => {
+  test('should render elements', async () => {
+    const indicator = 'test';
+    render(<SearchResults indicator={indicator} searchResults={[]} />);
+
+    expect(screen.getByRole('link')).toBeInTheDocument();
```

Comment:
Should we be using test IDs for all elements we look for in this page, rather than looking for text, etc.?

@williamwillis11, @alibombaye?